### DISCUSSION TITLE
LosslessCut vs. Avidemux

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#1916] [Support]
How do I make the interface of LosslessCut look similar to the one from Avidemux? Also moving frame-by-frame in the former is slower compared to the latter? How do I fix that?
![Screenshot (394)](https://github.com/mifi/lossless-cut/assets/141977822/4066e933-b651-48ff-9ca0-07b5df0ebb62)
![Screenshot (395)](https://github.com/mifi/lossless-cut/assets/141977822/393595c5-3181-4a1e-9934-cf61769db396)

**Comment by mifi** [comment]
losslesscut is a different app from avidemux so it cannot be changed to look like avidemux (unless you want to modify the source code yourself.)
